src/presenters/googlebot-presenter.php

```diff
@@ -20,6 +20,10 @@ class Googlebot_Presenter extends Abstract_Indexable_Presenter {
 	 * @return string The googlebot output tag.
 	 */
 	public function present() {
+		if ( $this->presentation->model->is_robots_noindex === true ) {
+			return '';
+		}
```

Comment:
Why did you put that here and not in the `Indexable_Presentation` or in the `get` method?

I would argue that the value itself is empty when no-index is set.

Reply:
There are multiple implementations for handling the robots noindex value and just one way to render this.